pytrnsys/psim/processTrnsysDf.py

```diff
@@ -398,13 +397,6 @@ def loadFiles(self):
             self.monDataDf["Cum_" + column] = self.monDataDf[column].cumsum()
         self.calcConfigEquations()
 
-        # This recalculated all, we should only recalculated what was done in caclConfigEquations. DC or it is so fast we don't care ?
```

Comment:
I'm trusting these are duplicated and not needed, then. We don't care about the "we should recalculate what was done in calcConfigEquations" part, then?